templates/messages_view.html

```diff
@@ -24,43 +24,16 @@
 
         <h1>View Messages</h1>
 
-        <!-- Example to delete -->
 
-        <p> This is an example </p>
+               {% for row in view %}
```

Comment:
Is there an alignment issue here?

Reply:
Hi @lioramilbaum 
Before we implemented the messages reading from the DB they created muck message to display.
Now we are reading the messages from the DB, so this code is redundant